src/utils/DateUtils.ts

```diff
@@ -1,24 +1,152 @@
 import { Options } from '../misc/Options';
 import { Utils } from './Utils';
 import { l } from '../strings/Strings';
-import { TimeSpan } from './TimeSpanUtils';
-import * as Globalize from 'globalize';
 import * as _ from 'underscore';
-
+import * as moment from 'moment';
+import * as d3 from "d3";
+import days = d3.time.days;
+/**
```

Comment:
I think we should add the `timeToString` method too since it also takes an `IDateToStringOptions` as a parameter:

> [ ... ] using the [`dateToString`]{@link DateUtils.dateToString}, [`timeToString`]{@link DateUtils.timeToString}, or [`dateTimeToString`] method [ ... ]